crates/re_log_types/src/absolute_time_range.rs

```diff
@@ -6,12 +6,12 @@ use crate::{NonMinI64, TimeInt, TimeReal};
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
-pub struct TimeRange {
+pub struct AbsoluteTimeRange {
```

Comment:
```suggestion
pub struct ResolvedTimeRange {
```
Absolute sounds like a time range that was provided by the user using absolute values as opposed to a range that has been resolved from a maybe-relative time-range.